docs/dev/wasm.md

```diff
@@ -50,75 +50,69 @@ and passing for each parameter a 64-bit value, of which top 32 bits are its size
 }
 ```
 
-## Support for NULL values
+## Parameter values
 
 Native WebAssembly types can only be represented directly if the function does not operate on NULL values. Fortunately, user-defined functions
 explicitly specify whether they accept NULL or not.
 
-If the function is specified not to accept NULL, all parameters are represented
-as in the description above.
+If the function is specified not to accept NULL, the parameter types natively supported by WASM are represented by their values (types shorter than 4 bytes are extended to 4 bytes),
+and all remaining types are represented as in the description above.
 
 If the function is specified to accept NULL, each parameter should be represented in WebAssembly by 
 size and pointer to its serialized form in memory, as in the paragraph above.
 
-the important distinction is that size equal to -1 (minus one)indicates that the value is NULL and should not be parsed.
+The important distinction is that size equal to -1 (minus one or i32::MAX) indicates that the value is NULL and should not be parsed.
```

Comment:
Is that not u32::MAX?

We shouldn't use rust terminology in wasm documentation, it's not limited to Rust, at least conceptually.

Reply:
Yes, it should be u32::MAX, or maybe, without using Rust terminology,  `0xffffffff`